docs/learn-balance-transfers.md

```diff
@@ -5,26 +5,51 @@ sidebar_label: How to transfer Balances
 ---
 
 Balance transfers are used to send one balance from one account to another account. To start
-transferring balances, we will begin by using the Polkadot-JS User Interface.
+transferring balances, we will begin by using [Polkadot-JS Apps][]. This guide assumes that you've
+already [created a Polkadot or Kusama account](learn-account-generation) and have some funds that
+will be transferred.
 
-## Polkadot-JS UI
+## Polkadot-JS Apps
 
-Begin by opening up the [Polkadot-JS UI](https://polkadot.js.org/apps). There are two ways to make a
-transaction: underneath "Accounts" by using the "Transfer" tab or through the "Accounts" tab. We
-will be starting off with the Accounts tab, so head there.
+Let's begin by opening up the [Polkadot-JS Apps][]. There are two ways to make a transaction:
 
-> Please note: For these examples, the network of choice is Polkadot. If you'd like to change this,
-> please click on the network selection from the top left corner of the navigation.
+1. By using the "Transfer" tab in the "Accounts" dropdown (located on the top navigational menu).
+2. Clicking the "send" button while in the "Accounts" page.
 
```

Comment:
Let's maybe add in a note here to be explicit on the network:
> Please note: For these examples, the network of choice is Polkadot. If you'd like to change this, please click on the network selection from the top left corner of the navigation.

Reply:
I agree with Kirsten's comment, other than that this looks good.